ext/web/06_streams.js

```diff
@@ -1138,7 +1829,12 @@
     assert(signal === undefined || signal instanceof AbortSignal);
     assert(!isReadableStreamLocked(source));
     assert(!isWritableStreamLocked(dest));
-    const reader = acquireReadableStreamDefaultReader(source);
+    let reader;
+    if (source[_controller] instanceof ReadableByteStreamController) {
+      reader = acquireReadableStreamBYOBReader(source);
```

Comment:
> If source.[[controller]] implements ReadableByteStreamController, let reader be either ! AcquireReadableStreamBYOBReader(source) or ! AcquireReadableStreamDefaultReader(source), at the user agent’s discretion.

I'll have to dig into that deeper. In what cases would a user agent use a DefaultReader?

Reply:
Not sure to be honest